code/modules/mob/emote.dm

```diff
@@ -107,6 +115,30 @@
 				span_notice("[flippy_mcgee] stumbles a bit after their flip."),
 				span_notice("You stumble a bit from still being off balance from your last flip.")
 			)
+*/
+/datum/emote/flip/check_cooldown(mob/living/carbon/user, intentional)
+	. = ..()
+	if(.)
+		return
+	if(!can_run_emote(user, intentional=intentional))
+		return
+	if(!iscarbon(user))
```

Comment:
I think the input arg needs to be mob/living/user, otherwise this if check always passes